c/src/ml-api-common.c

```diff
@@ -16,8 +16,33 @@
 #include <glib.h>
 
 #include "nnstreamer.h"
+#include "nnstreamer-tizen-internal.h"
 #include "ml-api-internal.h"
 
+/**
+ * @brief Gets the version number of machine-learning API.
+ */
+void
+ml_api_get_version (unsigned int *major, unsigned int *minor,
+    unsigned int *micro)
+{
+  if (major)
+    *major = VERSION_MAJOR;
+  if (minor)
+    *minor = VERSION_MINOR;
+  if (micro)
+    *micro = VERSION_MICRO;
+}
+
+/**
+ * @brief Gets the version string of machine-learning API.
```

Comment:
How about adding  "The returned string should be freed with g_free() when no longer needed." ?

Reply:
I added it in header, not .c.

Reply:
Thank you. I got it.